docs/.vuepress/config.js

```diff
@@ -128,5 +128,10 @@ module.exports = {
     searchMaxGuidesSuggestions: 5,
     searchMaxAPISuggestions: 10,
     fuzzySearchDomains: ['Core', 'Hooks', 'Options'],
+    // The list modifies the search results position. When the search phrase matches the pages
+    // below, the search suggestions are placed before the rest results. The pages declared in
+    // the array at the beginning have the highest display priority.
+    APISearchDomainPriorityList: ['Options'],
+    guidesSearchDomainPriorityList: [],
   }
```

Comment:
what do you think about moving all search focused options to one object like:
```js
search:{
    placeholder: 'Search...',
    guidesMaxSuggestions: 5,
    apiMaxSuggestions: 10,
    fuzzySearchDomains: ['Core', 'Hooks', 'Options'],
    // The list modifies the search results position. When the search phrase matches the pages
    // below, the search suggestions are placed before the rest results. The pages declared in
    // the array at the beginning have the highest display priority.
    apiSearchDomainPriorityList: ['Options'],
    guidesSearchDomainPriorityList: [],
}
```

Reply:
I'm ok with that 👍

Reply:
I'll prepare a fix

Reply:
Done 5a75498939a52c8681cd5c1eea52d8575e5080d6

Reply:
Thank you